nextjs-task-system/app/api/tasks/[id]/route.ts

```
@@ -0,0 +1,169 @@
+}
+
+export async function DELETE(
+  request: Request,
+  { params }: { params: { id: string } },
+) {
+  const { id } = params;
+
+  try {
+    const taskId = parseInt(id, 10);
+
+    if (isNaN(taskId)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)